appdata/network-manager-sstp.appdata.xml.in

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright Lubomir Rintel 2016 -->
+<component type="addon">
+  <id>network-manager-sstp</id>
+  <project_license>GPL-2.0+</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
That is just the licence for the AppStream XML file. The distributors typically bundle these snippets in their app store streams and a permissive license just avoids hassle there.

The tag above is about the project; still GPL.